site/content/docs/private-locations/checkly-agent-guide.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Installing Checkly Agent
+weight: 52
+menu:
+  docs:
+    parent: "Private Locations"
+aliases:
+- "/docs/private-locations/checkly-agent-guide/"
+---
+
+Note: The Checkly Agent currently only supports x86/AMD64 architecture and not ARM64. This means Apple Mac M1 systems are not currently supported.
+
+## About Checkly Agent
+
+**You must have the Owner or Admin role on your Checkly account to create private locations.**
+
+The Checkly Agent is a container-based tool that enables private locations in Checkly. The agent runs on your infrastructure in a container and runs checks on behalf of the Checkly application. Since the agent is local to your infrastructure, checks can run against any application accessible from the agent via HTTPS or HTTP.
+
+Here are the requirements before you get started:
+
+- The API key received when creating a private location in the Checkly account settings page
+- Container runtime (we test using Docker, but other OCI-compliant runtimes should work)
+- Outbound internet access to https://agent.checklyhq.com (proxy configuration is supported)
+- Access to your internal API- or browser-based application
+
+## Redundancy and scaling
+
+We recommend at least two agents per private location for redundancy. When considering scaling, you should use at least N+1 agents per private location to ensure checks will be processed even if an agent fails. Agents are stateless, ephemeral, and scale automatically. You can add additional agents to a location at any time and remove them as necessary. If an agent fails, the other agents in the same private location will continue to run the checks assigned to that location. Even checks that are currently in progress on an agent that fails will be rerun by another agent after a timeout period (140s). There are two cases where checks assigned to a location could fail to run:
```

Comment:
> Agents are stateless, ephemeral, and scale automatically.

I'm not sure if it's accurate to say that the Agent scales automatically. We can't start more Agent containers on the customers behalf, so it's up to them to handle scaling.

Reply:
Good point. Let me rephrase that a little.